clippy_lints/src/sig_drop_aux.rs

```diff
@@ -0,0 +1,60 @@
+// Auxiliary structures for lints involving `#[clippy::has_significant_drop]`.
```

Comment:
In a following PR the code declared here will be used in certain parts of the `significant_drop_in_scrutinee` lint

Reply:
NVM, for some reason it is not possible to create non-lint modules.